testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/ReloadRedirectTestCase.java

```diff
@@ -226,6 +226,7 @@ public void testReloadwithRedirect() throws Exception {
             promptFound = cliProc.pushLineAndWaitForResults("reload", "Accept certificate");
             assertTrue("No certificate prompt " + cliProc.getOutput(), promptFound);
         } finally {
+            cliProc.ctrlCAndWaitForClose();
             cliProc.destroyProcess();
```

Comment:
@jfdenise good point, I'll try on Windows and some exotic platforms we certify to be sure, and update the PR in case CtrlC is enough. Thank you.